### PR TITLE
MAUI-3684-Prepare UG for box and whisker series support in MAUI chart

### PR DESCRIPTION
**Pull request by NitheeshkumarThangaraj**
### Feature description ###
 
User Guide for the Box and whisker series in MAUI Platform

### Solution description ###
 
- I Created the UG Document in md Format named as BoxAndWhisker.md
- I attached a folder contains the Chart_types Images for Box and whisker series
- I attached the images to the md file
- I added the Box and whisker Series in the html List file

### Areas affected and ensured ###
 
None of the areas were affected and it was ensured.
 
### Behavioral Changes ###
 
None
 

### Does it have any known issues?
 
None
 
### Does it have memory leak?
 
It will be ensured.

### CODE REVIEW

File: MAUI/Cartesian-Charts/BoxAndWhisker.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
+---
+layout: post
+title: BoxAndWhisker Chart in .NET MAUI Chart control | Syncfusion
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/BoxAndWhisker.md:3]: Dont use API in all place.

File: maui-toc.html [hunk 2]
```diff
@@ -157,6 +157,7 @@
                     <li><a href="/maui/cartesian-charts/line">Line Chart</a></li>
                     <li><a href="/maui/cartesian-charts/fastline">FastLine Chart</a></li>
                     <li><a href="/maui/cartesian-charts/area">Area Chart</a></li>
+			  <li><a href="/maui/cartesian-charts/boxAndWhisker">BoxAndWhisker Chart</a></li>
```
Comment (Saravanan-Madhesh) [inline maui-toc.html:160]: change as Box and Whisker Chart
 "<a href="/maui/cartesian-charts/boxAndWhisker">Box and Whisker Chart</a>"

File: MAUI/Cartesian-Charts/BoxAndWhisker.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: Box and whisker Chart in .NET MAUI Chart control | Syncfusion
+description: Learn here all about BoxAndWhisker chart support in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/BoxAndWhisker.md:4]: At all place mention BoxAndWhisker  => Box and Whisker, mentioning API only use like above.

**Review by Saravanan-Madhesh — CHANGES_REQUESTED** [review]

File: MAUI/Cartesian-Charts/BoxAndWhisker.md (hunk 3 above)
Comment (NitheeshkumarThangaraj) [reply MAUI/Cartesian-Charts/BoxAndWhisker.md]: Okay. I have replaced

**Review by Saravanan-Madhesh — APPROVED** [review]

**Pull request merged**